### PR TITLE
SW-5370: Added application review by section screen

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2936?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2936** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2936?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2933** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2933?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx [hunk 1]
```diff
@@ -1,10 +1,233 @@
...
+          <ApplicationDeliverableRow title={strings.PROPOSED_PROJECT_BOUNDARY} goToDeliverable={() => {}} />
+
+          {sectionDeliverables(prescreenSection.moduleId).map((deliverable, index) => (
+            // Add link to deliverable
+            <ApplicationDeliverableRow
+              title={deliverable.name}
+              modifiedDate={
+                deliverable.modifiedTime ? DateTime.fromISO(deliverable.modifiedTime).toFormat('yyyy/MM/dd') : undefined
+              }
+              goToDeliverable={() => {}}
```
Comment (nickgraz) [inline src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx:197]: Is this intentional? Or maybe handled in a future PR?

File: src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx [hunk 2]
```diff
@@ -1,10 +1,233 @@
...
+    default:
+      return theme.palette.TwClrTxtWarning;
+  }
+};
+
+type ApplicationReviewProps = {
+  application: Application;
+};
+
+const ApplicationReview = ({ application }: ApplicationReviewProps) => {
```
Comment (nickgraz) [inline src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx:54]: Maybe this can go in its own file

**Review by nickgraz — APPROVED** [review]
Looks good! A couple minor things that could be improved on but not strictly necessary

File: src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx (hunk 1 above)
Comment (tommylau523) [reply src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx:197]: Yes we don't have deliverables route yet!

File: src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx [hunk 3]
```diff
@@ -1,10 +1,233 @@
...
+
+          {nonPrescreenSections.map((section) => (
+            <>
+              <Typography fontSize={'20px'} fontWeight={600} lineHeight={'28px'} marginTop={theme.spacing(3)}>
+                {section.name}
+              </Typography>
+
+              {sectionDeliverables(section.moduleId).map((deliverable, index) => (
+                // Add link to deliverable
+                <ApplicationDeliverableRow
```
[the remark this answers is not in the document]
Comment (tommylau523) [reply src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx:214]: We needed a way to distinguish application boundary row mostly. Not the most elegant way but it does need a different title, link and no modified time.

**Pull request merged**